databases/src/utils/openUrl.ts

```diff
@@ -0,0 +1,13 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as open from 'open';
+
+export async function openUrl(url: string): Promise<void> {
```

Comment:
IIRC there's some flavor of this in `vscode-azureextensionui` that can be used

Reply:
Tracking all suggestions for updating "create"/"utils" folder here: #1013